Youtube webpage project pull request for code review

javierbotero opened the pull request.

ivanid22 committed:
Initial commit

First commit after branching out of master, laying out main containers
for the page

javierbotero committed:
Added search bar and sign in section

-----------------------------

The sign in section couldn't be implemented entirely, for now isn't showing in the page, the search bar is showing.

ivanid22 committed:
added search button
--------------------
Added a seach button and adjusted some container elements

javierbotero committed:
Added the sections of the page

-----------------------------

Aside, Main, footer and the language section were added to the layout, backgrounds were added in order to recognize ho tThings were going.

ivanid22 committed:
Started working on language selection section
---------------------------------------------

Laid out the main structure for the language section container. Elements
still not placed.

javierbotero committed:
Add layout of viudeo thumbnails and video description, videio comments.

ivanid22 committed:
Started laying out the video info section
------------------------------------------
Placement for the data blocks is set, they still need to be filled.

javierbotero committed:
Added divs for sharing and other commands

___________----------___________________

Include span with button container but change the layout of the command container

ivanid22 committed:
Video commands continuation
---------------------------
Continued working on the command buttons, trying to align the button
image with the caption, without success.

javierbotero committed:
Added commands buttons below the video

----------------------------------------

Added the space of the commands and align them properly, added video link and added loading caption.

ivanid22 committed:
Added divisions on the container for the next video
---------------------------------------------------
Experimenting with flex as a display method on the new containers.

javierbotero committed:
Added The icons for user

------------------------

Using an anchor and divs the icons were put, but the play icon should be repositioned.

ivanid22 committed:
Started filling the suggested video box section
-----------------------------------------------

Setting containers, images and text in place for further styling in
another commit.

javierbotero committed:
Added susbcribe image and word

-------------

it was added an anchor, divs and span ir order to layout the information channel, the layout should be improved later.

ivanid22 committed:
Worked on video thumbnail containers, and started laying out the language selection bar

javierbotero committed:
Fixed position in the channel section

--------------------------

The position in channel section was aligned, using span for the text, and the anchor was positioned relative.

ivanid22 committed:
Finished language selection section. Small adjustments might be needed on fonts

javierbotero committed:
Added image icon in the new nav element

-------------------

Added the image navigation and the header has a nav element where all its elements are living there now

ivanid22 committed:
Laid out elements in the view count container area
--------------------------------------------------
Elements for the view count and view count bars set in place and
aligned.

javierbotero committed:
Added space evely between icons in the nav bar
---------------

Using flexbos in the container of the search bar and the icons they could be position evenly. It is neccesary to make littel adjustments.

ivanid22 committed:
Fixed the "up next" and "autoplay" sections
-------------------------------------------

Added a couple of new elements on the autoplay section, and fixed the
alignment on a few elements.

javierbotero committed:
Added font family rules and almost finished the nav bar
----------------------------
The fons were imported and the elements in the nav bar were position correctly, the element description was added but need completion.

ivanid22 committed:
Setting colors
----------------------------------
Giving most of the containers in the page their final colors. The
archive site is down at the moment, and it's making it difficult to do
actual work on the remaining sections.

javierbotero committed:
Organized text information and layout of that section

---------------

The padding was set for the container of the info

ivanid22 committed:
Adjusted some borders in the page
---------------------------------

Finished styling all the borders in the page and adjusted heights on some of the containers.

javierbotero committed:
Added better layou for info video section

----------------

Improvements in the spaces between the icons in the video information section

ivanid22 committed:
Working on the subsections of the footer
-----------------
The configuration section has also been filled with its corresponding buttons and logo.

javierbotero committed:
Added Policy Menu

--------------

The Menu was added and styled using ul and buttons

ivanid22 committed:
Finished footer section

--------------------------

Gave the final touches to the footer section, styled elements appropriately

ivanid22 committed:
Started validating code

javierbotero committed:
Document finished and completed with the mark-up checker

pelzolga123 commented on css/styles.css:
```diff
@@ -0,0 +1,723 @@
+
+#container {
+    width: 1024px;
+    height: 3000px;
```
If you remove height value, empty space after footer will disappear.

pelzolga123 commented on index.html:
```diff
@@ -0,0 +1,347 @@
+                                <a href="https://youtube.com" title="Youtube Home">
+                                    <span title="Youtube Home" id="logo"></span>
+                                    <span id="logo-ornament">NL</span>
+                                </a>
```
Image should be embedded inside img tag.

pelzolga123 reviewed (changes requested):
Project requires some changes and descriptive Readme file.

pelzolga123 commented:
## Hi Team! ![](https://user-images.githubusercontent.com/38801423/62908122-703d4480-bd44-11e9-9115-42025ab7f341.png) 
#### Your project looks good!
* [x] Image should be embedded inside img tag not added from css
[Link to requirement screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/navbar_logo_requirement.png)
* [x] Project should have at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar.
[Link to requirement screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/icons_requirement.png)
* [x] Project should have like and dislike icons
[Link to requirement screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_like_requirement.png)
* [x] 'Share' should be a button.
[Link to requirement screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png)
* [x] Project should have a descriptive README.md
[Link to requirement screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png)
* Also this part looks not very good, it would look better if you remove empty space after footer:
![image](https://user-images.githubusercontent.com/20416431/69889019-76593000-12f7-11ea-97b8-aeee6c151098.png)

### Please kindly Fix the above issue and send another code review request.

If you have any questions or need help, you can message me on slack. *Happy coding*

ivanid22 committed:
Fixing issues related to our code review
-----------------------------------------

Fixed 4 of the issues with our pull request, there's still one left.

javierbotero committed:
Finished with the required Changes for the pull request
____________________________________________

Added the images on the right most part of the nav-bar

shshamim63 commented:
## Status :red_circle:  [!! Required Changes !!] :red_circle: 
___
Hi @javierbotero  and @ivanid22  :wave: ,
Awesome job :clap: . Really a good job as a first project of this HTML & CSS section. Most of the items are rendering at the place it should. Code looks neat and clean. But there are some issue that are actually blocking to declare this project as complete. 
___
##### Please fix the issues below :point_down: to complete this project.
___
### :spades: [Mandatory Changes] :spades: 
___
1. You have to Check HTML markup with this validator https://validator.w3.org/#validate_by_upload.
![image](https://user-images.githubusercontent.com/38625956/69891728-ea5ffc00-1328-11ea-8d2d-136c8ea23351.png)

I have done it. Found some errors. Please fix them.
![image](https://user-images.githubusercontent.com/38625956/69891617-fb5c3d80-1327-11ea-9586-e995a1783eab.png)

2. The readme is not well descriptive. You can make it more descriptive by answering some questions. Like->
..1. What is this project about?
..2. What have you learned?
..3. Is there any online demo.
..4. Who are the contributors?
You can decorate by using this [link](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
___
##### Please fix the above mentioned issue and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](Shakhawat Hossain)

ivanid22 committed:
Fixed issues with our last code review.
---------------------------------------
Readme file expanded to be more descriptive and ran a syntax checkup.

nikasulo commented:
# :smile: you've done it! @ivanid22 and @javierbotero 
Great work  handling the errors and creating the readme :+1: . Go ahead and mergeeeeeeeee :rocket: .
